### PR TITLE
[EMB-138]New WG1000 UI content update

### PR DESCRIPTION
**Pull request by liliumjsn**
# New WG1000 UI content update
- Updated product photos for WS1000 and WS2000
- Updated app themed screenshots for wallet connect
- Created tabs in WS1000 connect and claim process
- Updated M5 UI content in WS1000 product

### CODE REVIEW

File: docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx [hunk 1]
```diff
@@ -5,19 +5,36 @@ sidebar_label: Connect and Claim Station
 slug: /wxm-devices/wifi-m5-bundle/m5-connect-and-claim
 ---
 
+import Tabs from "@theme/Tabs";
 import useBaseUrl from "@docusaurus/useBaseUrl";
 
+## M5 Miner
+<Tabs groupId="wg1000_ui">
+  <TabItem value="wg1000_ui_new" label="UI versions > 0.10.0">
+  <img object-fit="contain" src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/m5-ui-new.svg")} />
```
Comment (gkoumanos) [inline docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx:14]: Not a big fan of svg files for web. Just make sure they do not appear huge or break layout as it happens in some pages where huge svg images are used

File: static/img/home/helium.png
Comment (gkoumanos) [inline static/img/home/helium.png:1]: Do we need that huge of a resolution and filesize (1.3MB)?

File: docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx (hunk 1 above)
Comment (gkoumanos) [inline docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx:14]: On my (local) computer the image appears with a few sec delay. Possibly trying to render all its path elements

File: docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx [hunk 2]
```diff
@@ -164,16 +220,24 @@ Once the WiFi credentials have been successfully transferred to M5 Miner, the de
 ### Type the M5 Miner serial number when prompted
 
 
-<div class="explanation small-big">
-  <img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/serial-number.jpg")} />
-  <span>
+<div class="explanation">
+        <img style={{width:"calc(100%*1/3 - 10px)"}} src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/serial-number.jpg")} />
```
Comment (gkoumanos) [inline docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx:224]: Screw you CSS, I'll do it my way! Does it work ok on mobile view and iPhones?

File: docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx [hunk 3]
```diff
@@ -92,12 +117,26 @@ If unsure, check the sticker under the station or the box that came in it.
...
-  <img class="control" src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/m5-change-frequency.svg")} />
-  <ol>
-    <li>Swipe left on the touch screen</li>
-    <li>Click the Frequency options and select the new frequency your M5 Miner will listen to</li>
-    <li>A confirmation dialog will open. Read it carefully and click "Yes" or "No" accordingly</li>
-  </ol>
+  
+  <Tabs groupId="wg1000_ui">
+  <TabItem value="wg1000_ui_new" label="UI versions > 0.10.0">
+    <img class="control" src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/m5-change-frequency-new.svg")} />
```
Comment (gkoumanos) [inline docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx:123]: 17MB file. Please use PNG instead

File: docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx [hunk 4]
```diff
@@ -92,12 +117,26 @@ If unsure, check the sticker under the station or the box that came in it.
...
+    <ol>
+        <li>Swipe right on the touch screen</li>
+        <li>Click the Frequency options and select the new frequency your M5 Miner will listen to</li>
+        <li>A confirmation dialog will open. Read it carefully and click "Continue" or "Cancel" accordingly</li>
+    </ol>
+  </TabItem>
+  <TabItem value="wg1000_ui_old" label="Older UI versions">
+    <img class="control" src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/m5-change-frequency.svg")} />
```
Comment (gkoumanos) [inline docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx:131]: 17MB file. Please use PNG instead. My PC cannot do for some reason

File: docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx [hunk 5]
```diff
@@ -108,15 +147,32 @@ If you don't use the correct frequency as your weather station, M5 Miner will no
 
 ### Connect M5 Miner to WiFi
 
-<img class="marginTop" src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/m5-configure-wifi.svg")} />
+<Tabs groupId="wg1000_ui">
+  <TabItem value="wg1000_ui_new" label="UI versions > 0.10.0">
+    <img class="marginTop" src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/m5-configure-wifi-new.svg")} />
```
Comment (gkoumanos) [inline docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx:152]: 17MB file. Please use PNG instead. My PC cannot do for some reason

File: docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx [hunk 6]
```diff
@@ -108,15 +147,32 @@ If you don't use the correct frequency as your weather station, M5 Miner will no
...
+
+By now, your M5 Miner is set and ready for claiming!
+Swipe right to navigate to the settings screen. There, tap on "Configure WiFi" to switch to connect mode.
+
+Once this is done, connect to M5 Miner via WiFi (you should see in the networks screen a "WXMMinerM5").
 
+A popup will open, asking you to select the desired network you'd like to connect the M5 Miner. Select the one you want and provide the password for it.
+  </TabItem>
+  <TabItem value="wg1000_ui_old" label="Older UI versions">
+    <img class="marginTop" src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/m5-configure-wifi.svg")} />
```
Comment (gkoumanos) [inline docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx:162]: 17MB file. Please use PNG instead. My PC cannot do for some reason

File: docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx [hunk 7]
```diff
@@ -164,16 +220,24 @@ Once the WiFi credentials have been successfully transferred to M5 Miner, the de
...
+  <div style={{width:"calc(100%*2/3 - 10px)"}} >
 
-#### Navigate to M5 Miner Settings and find the device's Serial Number
+#### Navigate to M5 Miner and find the device's Serial Number
 
 Follow the steps shown in the image below to find your M5 Miner's Serial Number. You'll need to type this in the WeatherXM app in order to verify that you own this station and secure it for yourself.
+    
+<Tabs groupId="wg1000_ui">
+  <TabItem value="wg1000_ui_new" label="UI versions > 0.10.0">
+    <img object-fit="contain" src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/m5-serial-new.svg")} />
```
Comment (gkoumanos) [inline docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx:233]: 17MB file. Please use PNG instead. My PC cannot do for some reason

File: docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx [hunk 8]
```diff
@@ -164,16 +220,24 @@ Once the WiFi credentials have been successfully transferred to M5 Miner, the de
...
+  </TabItem>
+  <TabItem value="wg1000_ui_old" label="Older UI versions">
+    <img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/m5-serial.svg")} />
```
Comment (gkoumanos) [inline docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx:236]: 17MB file. Please use PNG instead. My PC cannot do for some reason

File: docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx (hunk 2 above)
Comment (liliumjsn) [reply docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx:224]: I gave up trying to make M5's look bigger.

**Review by gkoumanos — APPROVED** [review]
LGTM, didn't build last 2 commits.

**Pull request merged**